ofl/agbalumo/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1,7 @@
+<p>Curvy, chunky, and compact. The Agbalumo display typeface has been designed to represent and capture the beauty of African languages. Primarily taking shape from the use of a brush pen, it's charming, playful, and cute look lends itself to a variety of commercial use cases. From beautifully curated eye catching websites to brightly colored packaged items, if you aim to communicate personality, Agbalumo is a sure selection.</p>
+
+<p>Agbalumo is a single weight font. The glyph set includes standard opentype features and an assortment of stylistic alternates. Agbalumo can be used for African languages that make use of the Latin script, European languages, and Vietnamese.</p>
+
+<p>Agbalumo is designed by <a href="https://www.instagram.com/columntype">ColumnType</a></p>
```

Comment:
@eliheuer the Instagram page should be part of the author's biography. This form should be fill up by the author: https://docs.google.com/forms/d/e/1FAIpQLSeMwHN8J213ZaxHrr5lHCrX56HY_NjGrWB8o604g98YxuMrdA/viewform. It is linked to a spreadsheet I shared with you. This article will help to add the bio: https://googlefonts.github.io/gf-guide/profile.html.

Having ColumnType in the description and Raphael Alegbeleye in the metadata.pb is a bit confusion. The author should decide on how they want to be credited.

Reply:
The Instagram link was fixed in this PR: https://github.com/SorkinType/Agbalumo/pull/16

I talked with the designer about the designer profile, and he said he completely filled out the form. I'm confirming I see his info in the Designer Profile Info sheet.